keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/nuts/Stepsize.java

```diff
@@ -104,10 +107,10 @@ public static double findStartingStepSize(Map<VertexId, DoubleTensor> position,
      */
     public double adaptStepSize(Tree tree, int sampleNum) {
 
-        double logStepSize = logStepSizeFrozen;
-
         if (sampleNum < adaptCount) {
             logStepSize = updateLogStepSize(tree, sampleNum);
+        } else {
```

Comment:
Good spot. I thought we were meant to return the last calculated value of `logStepSize` here